SVGBezierPath.mm

```diff
@@ -72,8 +72,11 @@ + (NSArray *)pathsFromSVGString:(NSString * const)svgString
 
 - (NSString *)SVGRepresentation
 {
-    return SVGStringFromCGPaths(@[(__bridge id)self.CGPath], nil);
+    SVGMutableAttributeSet *attributes = [SVGMutableAttributeSet new];
+    [attributes setAttributes:self.svgAttributes forPath:self.CGPath];
+    return SVGStringFromCGPaths(@[(__bridge id)self.CGPath], [attributes copy]);
```

Comment:
Wouldn't `return SVGStringFromCGPaths(@[(__bridge id)self.CGPath], self.attributes)` work?

Reply:
Unfortunately, it wouldn't work because `self.attributes` is a `NSDictionary<NSString*, id>` and  `SVGStringFromCGPaths` expects `SVGAttributeSet` as the second parameter.

Reply:
Ah, ok. Sorry. It's been a little while since I worked on this :)
Then apart from the copy being unnecessary, it looks good!